examples/spin-timer/readme.md

```diff
@@ -12,3 +12,22 @@ To test:
 * `spin plugin install --file ./trigger-timer.json --yes`
 
 Then you should be able to `spin build --up` the [guest](./app-example/).
+
+## Pluginify
+
+To test:
+* create a `spin-pluginify.toml` file as follows:
+```toml
+name= "trigger-timer"
+description= "IT IS I THE GREAT OZ"
+homepage= "www.example.com"
+version= "0.1.0"
+spin_compatibility= ">=0.7"
```

Comment:
This probably ought to be `>=2.0` at this point though it doesn't much matter for the local dev case

Reply:
I changed the description, spin_compatibility, and homepage in both files `REAME.md` and `trigger-timer.json`, right?

Reply:
Perfect - thanks!

Reply:
To clarify, are these changes in progress or are they pushed?  There's no hurry of course, just checking my assumptions!

Reply:
Good point, should have been pushed.Good point, should have been pushed. But weren't.